test/types/types.test.ts

```diff
@@ -45,7 +45,7 @@ const options2 = {
 
 const options3 = {
   global: true,
-  max: (req: FastifyRequest<RequestGenericInterface>, key: string) => (42),
+  max: (req: FastifyRequest<RequestGenericInterface>, key: string) => Promise.resolve(42),
```

Comment:
Could you add a `option4` object: changing this one we are no more testing the number

Reply:
Is there a need for an options4, it seems like options 2 already tests the number to me
https://github.com/fastify/fastify-rate-limit/blob/ded16faae4ae4e71b499a108dbab9002e96818af/test/types/types.test.ts#L39-L44

Reply:
I did change it to what you asked, if there was a reason I'm just not seeing